### PR TITLE
feat: #1864 Update organisation tab in dev portal

### PR DESCRIPTION
**Pull request by nphivu414**
# Pull request checklist

## Does this close any currently open issues?

fixes: #1864 

**Please check if your PR fulfills the following requirements:**

- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been reviewed and added / updated if needed (for bug fixes / features)
- [x] Build (`yarn build`) was run locally and any changes were pushed
- [x] Lint (`yarn lint`) has passed locally and any fixes were made for failures
- [x] Test (`yarn test`) has passed locally and any fixes were made for failures

## Pull request type

**Please check the type of change your PR introduces:**

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## What is the current behavior?

Issue Number: #1864 

## What is the new behavior?

- Add ‘No VAT Number’ checkbox and ‘No Company Registration Number’ checkbox

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

## Other information

image

### CODE REVIEW

File: packages/marketplace/src/components/pages/settings/developer-settings-organisation-tab/organisation-form/form-schema/form-fields.ts [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+    name: 'countryId',
+    label: 'Country',
+  },
+  iconImageUrlField: {
+    name: 'iconImageUrl',
+    label: 'Upload Logo',
+  },
+  line1Field: {
+    name: 'line1',
+    label: 'Line 1*',
```
Comment (willmcvay) [inline packages/marketplace/src/components/pages/settings/developer-settings-organisation-tab/organisation-form/form-schema/form-fields.ts:78]: The address fields are now Address Line 1, Address Line 2, Address Line 3 and Address Line 4. Postcode etc remain the same

File: packages/marketplace/src/components/pages/settings/developer-settings-organisation-tab/organisation-form/organisation-form.tsx [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    >
+      {({ values }) => {
+        return (
+          <Form>
+            <H3 isHeadingSection>Company Information</H3>
+            <FormSection>
+              <CompanyInformationSection formValues={values} />
+              <CompanyAddressSection />
+              <LevelRight>
+                <Button className="mt-8" type="submit">
```
Comment (willmcvay) [inline packages/marketplace/src/components/pages/settings/developer-settings-organisation-tab/organisation-form/organisation-form.tsx:48]: That mt-8 class doesn't do anything -  you can remove, the grid above handles that vertical spacing.

File: packages/marketplace/src/components/pages/settings/developer-settings-organisation-tab/organisation-form/company-address-section.tsx [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+import { Grid, Input, GridItem, SelectBox } from '@reapit/elements'
+import { formFields } from './form-schema/form-fields'
+
+export type CompanyAddressSectionProps = {}
+
+const { line1Field, line2Field, line3Field, line4Field, countryIdField, postCodeField } = formFields
+
+const CompanyAddressSection: React.FC<CompanyAddressSectionProps> = () => {
+  return (
+    <>
```
Comment (tanphamhaiduong) [inline packages/marketplace/src/components/pages/settings/developer-settings-organisation-tab/organisation-form/company-address-section.tsx:11]: if you use Grid you can remove this

File: packages/marketplace/src/components/pages/settings/developer-settings-organisation-tab/organisation-form/company-information-section.tsx [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+} = formFields
+
+const CompanyInformationSection: React.FC<CompanyInformationSectionProps> = ({ formValues }) => {
+  const { noVatNumber, noReg } = formValues
+  const isVATNumberFieldDisabled = noVatNumber
+  const isRegFieldDisabled = noReg
+  const isNationalInsuranceNumberFieldVisible = isVATNumberFieldDisabled || isRegFieldDisabled
+
+  return (
+    <>
```
Comment (tanphamhaiduong) [inline packages/marketplace/src/components/pages/settings/developer-settings-organisation-tab/organisation-form/company-information-section.tsx:30]: you use Grid, you can remove it

File: packages/marketplace/src/components/pages/settings/developer-settings-organisation-tab/organisation-form/company-address-section.tsx (hunk 3 above)
Comment (nphivu414) [reply packages/marketplace/src/components/pages/settings/developer-settings-organisation-tab/organisation-form/company-address-section.tsx:11]: Hi @tanphamhaiduong ,
I need to return multiple Grid components here so they should be wrapped in a Fragment

File: packages/marketplace/src/components/pages/settings/developer-settings-organisation-tab/organisation-form/company-information-section.tsx (hunk 4 above)
Comment (nphivu414) [reply packages/marketplace/src/components/pages/settings/developer-settings-organisation-tab/organisation-form/company-information-section.tsx:30]: Same as above ^

**Pull request merged**